rest/rest-api/src/main/java/org/apache/brooklyn/rest/api/ActivityApi.java

```diff
@@ -75,6 +75,7 @@ public TaskSummary get(
     @ApiResponses(value = {
             @ApiResponse(code = 404, message = "Could not find task or stream")
     })
+    @Produces(MediaType.TEXT_PLAIN + ";qs=0.9")
```

Comment:
why the `qs`? it's not actually a valid parameter on `Content-Type`, it's some sort of Apache webserver hack.